src/App.js

```diff
@@ -0,0 +1,189 @@
+  constructor() {
+    super();
+    this.playerRef = React.createRef();
+    this.graphRef = React.createRef();
+    this.sdkRef = React.createRef();
+
+    this.state = {
+      buffer: 0,
+      latency: 0,
+      engineName: "",
+      playbackState: "",
+      quality: "",
+      version: window.exmg.livery.version,
+      config:
+        "https://cdn.playtotv.com/video-encoder/remoteconfigs/5ddb98f5e4b0937e6a4507f2.json",
+    };
+  }
+
+  componentDidMount() {
+    this.player = this.playerRef.current;
+    this.player.addEventListener("livery-time-update", (e) => {
+      this.updateBufferLatency();
+    });
+    this.player.addEventListener("livery-started", (e) => {
+      this.updateEngineName();
+      this.updateQuality();
+    });
+    this.player.addEventListener("livery-playback-change", (e) => {
+      this.updatePlaybackState();
+    });
+    this.player.addEventListener("livery-active-quality-change", (e) => {
+      this.updateQuality();
+    });
+
+    const graph = this.graphRef.current;
+    graph.player = this.player;
+    graph.backgroundColor = "transparent";
+    graph.textColor = "white";
+    graph.bufferColor = "deeppink";
+    graph.latencyColor = "yellow";
+    graph.height = "100%";
+    this.setStream();
+  }
+
+  updateBufferLatency() {
+    let state = this.state;
+    state.buffer =
+      Math.round((this.player.buffer + Number.EPSILON) * 100) / 100;
+    state.latency =
+      Math.round((this.player.latency + Number.EPSILON) * 100) / 100;
+    this.setState(state);
+  }
+
+  updateEngineName() {
+    let state = this.state;
+    state.engineName = this.player.engineName.replace("Engine", "");
+    this.setState(state);
+  }
+
+  updatePlaybackState() {
+    let state = this.state;
+    state.playbackState = this.player.playbackState;
+    this.setState(state);
+  }
+
+  updateQuality() {
+    const {
+      activeQuality: activeIndex,
+      selectedQuality: selectedIndex,
+      qualities,
+    } = this.player;
+
+    const active = Number.isNaN(activeIndex) ? null : qualities[activeIndex];
+    const selected = Number.isNaN(selectedIndex)
+      ? null
+      : qualities[selectedIndex];
+
+    let selectedStr = "";
+    if (qualities.length > 1) {
+      if (selected) {
+        if (!active || selectedIndex !== activeIndex) {
+          selectedStr = `=> ${selected.label}`;
+        }
+      } else {
+        selectedStr = "(auto)";
+      }
+    }
+    let quality = `${active ? active.label : ""} ${selectedStr}`;
+    let state = this.state;
+    state.quality = quality;
+    this.setState(state);
+  }
+
+  changeLogLevel(event) {
+    this.sdkRef.current.logLevel = event.target.value;
+  }
+
+  setStream() {
+    let params = new URLSearchParams(window.location.search);
+    let streamID = params.get("stream");
+
+    if (!streamID) {
+      streamID = "5ddb98f5e4b0937e6a4507f2";
+    }
+    let config = this.getCustomerConfig(streamID);
+    let source = this.getCustomerSource(streamID);
```

Comment:
This and the getCustomerSource() method are no longer used and should be removed.